src/Test.UnitTests.Sarif.Multitool.Library/ApplyPolicyCommandTests.cs

```diff
@@ -43,7 +43,8 @@ private SarifLog ExecuteTest(string path)
             {
                 InputFilePath = path,
                 OutputFilePath = path,
-                Force = true
+                Force = true,
+                SarifOutputVersion = SarifVersion.Current
```

Comment:
>SarifOutputVersion = SarifVersion.Current [](start = 16, length = 41)

should we add this as default value in the options itself?

Reply:
Done.

---
In reply to: [564751819](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r564751819) [](ancestors = 564751819)